CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## 0.23.2 (Unreleased)
 
+### Important
+
+This release changes the behavior of the provider namespace flag introduced in `0.23.0`.
+Previously, this flag was treated as an override, which ignored namespace values set
+directly on resources. Now, the flag is a default, and will only set the namespace if one
+is not already set. **This may cause resources to be recreated!**
+
```

Comment:
Should we phrase this as `BREAKING` as in the entry below?

> provider namespace flag

Should `namespace` be quoted to be a little clearer this refers to the `namespace` flag property (or `kubernetes:namespace` config variable)?  

> **This may cause resources to be recreated!**

I'm not sure the bold an exclamation are necessary.  We should be matter of fact and clear about the impact.  We should also qualify this to "deployments which used this property ...", since that is the significant minority of users.

Reply:
Updated on #587